gprofiler/utils/perf.py

```diff
@@ -44,3 +69,51 @@ def valid_perf_pid(pid: int) -> bool:
     try and look up the processes related to them.
     """
     return pid not in (0, -1)
+
+
+def _collapse_stack(comm: str, stack: str, insert_dso_name: bool = False) -> str:
```

Comment:
Exported stuff - don't prefix with `_`

Reply:
IDM, I wanted this PR to be a copy (except imports let's say)